### DISCUSSION TITLE
how to add one custom mail header,thanks

### DISCUSSION THREAD

**Discussion by ryuushihou** [Mailu/Mailu#3214] [User support]
After I built an email server using mailu, although I can send and receive emails normally, I want to add a custom header information to the email. Is there any relevant document that I can refer to?

i had tried this： but it cant work..  

![image](https://github.com/Mailu/Mailu/assets/113272024/83ec9aed-532d-4c97-aea8-4d00da83ba54)
![image](https://github.com/Mailu/Mailu/assets/113272024/f516405e-1c11-44c7-8f72-f420f83df3d3)

**Comment by Rytr-0** [comment]
Hi, did you find a solution for this yet? I'm having the same issue trying to implement an n-subscribe header.
